.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,18 @@
+name: Release-drafter
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      # Drafts the next Release notes as Pull Requests are merged into "master"
+      - uses: release-drafter/release-drafter@v5
+        with:
+          config-name: release-drafter-config.yml
```

Comment:
rename to `release-notes-drafter-config.yml`

Reply:
Done.